RULES.md

```diff
@@ -125,6 +125,7 @@ Inspections Lists (Language level migration)
 | Language level migration | PowerOperatorCanBeUsedInspection                | Power operator can be used                          | yes | yes | no   | no  |
 | Language level migration | ConstantCanBeUsedInspection                     | A constant can be used                              | yes | yes | yes  | no  |
 | Language level migration | ArgumentUnpackingCanBeUsedInspection            | Argument unpacking can be used                      | yes | yes | no   | no  |
+| Language level migration | ProtectedWithFinalClassInspector                | Protected modifier could be replaced by private     | no  | yes | no   | no  |
```

Comment:
I can move that to Code Style, but I don't agree with this class name because it treats the "protected" keyword used inside "final" classes (then, "protected with final class"), and  your suggestion seems operates over the "final" keyword ("Unnecessary final modifier"). Can you rethink about that and tell me if it is reality what we should do?